packages/wrangler/src/__tests__/constellation.test.ts

```diff
@@ -75,7 +76,7 @@ describe("constellation commands", () => {
 
 	beforeEach(() => {
 		// @ts-expect-error we're using a very simple setTimeout mock here
-		jest.spyOn(global, "setTimeout").mockImplementation((fn, _period) => {
+		vi.spyOn(global, "setTimeout").mockImplementation((fn, _period) => {
```

Comment:
We use this pattern in a bunch of places—we should refactor to use Vitest fake timers at some point. I didn't do that here because I wanted to keep the diff clean